### PR TITLE
Update Configuration.java

### PR DESCRIPTION
**Pull request by davidgraeff**
Change class javadoc. The class is not only for Things.

Signed-off-by: David Gräff <david.graeff@web.de>

### CODE REVIEW

File: bundles/org.openhab.core.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java [hunk 1]
```diff
@@ -27,7 +27,11 @@
 import org.eclipse.smarthome.config.core.internal.ConfigMapper;
 
 /**
- * This class is a wrapper for configuration settings of {@link Thing}s.
+ * {@link Configuration} wraps String->Object associations and allows to map
```
[the remark this answers is not in the document]
Comment (maggu2810) [reply bundles/org.openhab.core.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java]: What is meaned by `String->Object associations`? A collection of string keys and object values (`Map<String,Object>`)?

File: bundles/org.openhab.core.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java [hunk 2]
```diff
@@ -27,7 +27,11 @@
...
+ * such an associations to java classes via the {@link Configuration#as} method.
+ * <p>
+ * Use {@link Configuration} for {@link Thing} settings as well as OSGi service
```
Comment (maggu2810) [inline bundles/org.openhab.core.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java]: Perhaps "Configuration can be used for ..."?

File: bundles/org.openhab.core.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidgraeff) [reply bundles/org.openhab.core.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java]: Yes. A java map is a key value association in general isn't it. But if you have a better wording, that is welcome of course.

I didn't want to use the word map as that is associated with the Java class and a java Dictionary is also compatible.

[the remark this answers is not in the document]
Comment (maggu2810) [reply bundles/org.openhab.core.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java]: Wording ;)

Perhaps:
... wraps a collection of key-value-pairs (string keys and values of any type) and allow ...

Or leave it as it is but I still think you need to replace the `>` character

```
entities for the less than symbol (<) and the greater than symbol (>) should be written as &lt; and &gt;. Similarly, the ampersand (&) should be written as &amp;
```

What about "wraps String to Object associations"?

**Comment by maggu2810** [comment]
No interest anymore?

**Comment by davidgraeff** [comment]
I will surely come across the annoyance a second time and open a PR again. This one was a github web PR, where changes cannot be applied easily without breaking the DCO and signed-off-by stuff.

**Comment by openhab-5iver** [comment]
I started using VS Code with a clone of openhab-docs and it has made doc changes even easier than using the GH website. And git can be used right in the terminal to amend commits.

**Pull request closed without merging**